bot/cogs/snakes.py

```diff
@@ -303,85 +309,78 @@ def event_check(reaction_: Reaction, user_: Member):
             """
             Make sure that this reaction is what we want to operate on
             """
-
-            no_restrictions = (
-                user_.id == ctx.author.id
-            )
-
             return (
                 # Conditions for a successful pagination:
                 all((
-                    # Reaction is on this message
-                    reaction_.message.id == board_id.id,
-                    # Reaction is one of the pagination emotes
-                    reaction_.emoji in ANTIDOTE_EMOJI,
-                    # Reaction was not made by the Bot
-                    user_.id != self.bot.user.id,
-                    # There were no restrictions
-                    no_restrictions
+                    reaction_.message.id == board_id.id,  # Reaction is on this message
+                    reaction_.emoji in ANTIDOTE_EMOJI,  # Reaction is one of the pagination emotes
```

Comment:
best to line these comments up with each other, much better readability. PEP8 only says there should be _at least_ 2 spaces, but does not say anything about not being allowed to use five if it helps your program become more readable. so this:

```py
something = something      # do a useless thing
something = not something  # oh okay why did we even do the previous thing then
```

is encouraged.